Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM rust:latest as builder
+ENV APP aderyn
+WORKDIR /usr/src/$APP
+COPY . .
+RUN cargo install --path ./aderyn
+ 
+FROM debian:bookworm-slim
+RUN apt-get update && rm -rf /var/lib/apt/lists/*
+COPY --from=builder /usr/local/cargo/bin/$APP /usr/local/bin/$APP
+WORKDIR /share
```

Comment:
🤔 Umm I don't see forge and hardhat here.... can you explain how this would work without me having to install those 2?

Reply:
Do you want to maybe take some help from here ? 😉

```docker
# Use the latest foundry image
FROM ghcr.io/foundry-rs/foundry

# Copy our source code into the container
WORKDIR /app

# Build and test the source code
COPY . .
RUN forge build
RUN forge test
```
Source: https://book.getfoundry.sh/tutorials/foundry-docker?highlight=docker#dockerizing-a-foundry-project